modules/extensions/galasa-extensions-parent/dev.galasa.ras.couchdb/src/main/java/dev/galasa/ras/couchdb/internal/CouchdbDirectoryService.java

```diff
@@ -399,13 +386,15 @@ private RasRunResultPage getRunsPageFromCouchdb(HttpPost httpPost, Find query) t
             for (TestStructureCouchdb ts : found.docs) {
                 if (ts.isValid()) {
 
-                    // Don't load the artifacts for the found runs, just set a root location for artifacts
+                    // Don't load the artifacts for the found runs, just set a root location for
+                    // artifacts
```

Comment:
Were these comments auto-formatted? Seems a bit weird to only have a single word on a line followed by a continuation on the next line